scripts/set_nixpath.sh

```diff
@@ -0,0 +1,6 @@
+update_NIX_PATH() {
+  local scriptDir=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+  export NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/$(nix-shell -p jq --run "jq .rev < ${scriptDir}/../nixpkgs-src.json -r").tar.gz
+  export NIX_PATH_LOCKED=1
```

Comment:
OK, please add a comment so that from the code one can get that, otherwise the next person might delete it assuming it was forgotten to be removed.